File: app/src/main/java/com/siju/acexplorer/ui/peekandpop/DimensionUtil.java
package com.siju.acexplorer.ui.peekandpop;

import android.content.Context;
import android.support.annotation.NonNull;
import android.util.TypedValue;

/**
 * Created by Vincent on 30/01/2016.
 */
public class DimensionUtil {

    public static int convertDpToPx(@NonNull Context context, int dp) {
        return (int) TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, dp, context.getResources().getDisplayMetrics());
    }


}

File: app/src/main/java/com/siju/acexplorer/ui/peekandpop/DimensionUtil.kt
package com.siju.acexplorer.ui.peekandpop

import android.content.Context
import android.util.TypedValue

/**
 * Created by Vincent on 30/01/2016.
 */
object DimensionUtil {

    fun convertDpToPx(context: Context, dp: Int): Int {
        return TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, dp.toFloat(), context.resources.displayMetrics).toInt()
    }
}